example/lib/main.dart

```diff
@@ -81,6 +81,47 @@ class HomePage extends StatelessWidget {
               ),
             ),
           ),
+          Card(
+            child: Padding(
+              padding: EdgeInsets.all(8),
+              child: Column(
+                children: <Widget>[
+                  Text(
+                    'Expand Child Horizontally',
+                    style: Theme.of(context).textTheme.headline6,
+                  ),
+                  SizedBox(height: 8),
+                  Row(
```

Comment:
What's changed here is basically just adding Row instead of Column, and setting vertical to false